colors/railscasts.vim

```diff
@@ -38,7 +38,7 @@ hi StatusLine                guibg=#414243 gui=NONE guifg=#E6E1DC
 hi StatusLineNC              guibg=#414243 gui=NONE
 hi VertSplit                 guibg=#414243 gui=NONE guifg=#414243
 hi CursorLineNr              guifg=#f4f800 ctermfg=248
-hi ColorColumn               guibg=#333435 ctermbg=235
+hi CursorColumn              guibg=#333435 ctermbg=235 cterm=NONE
```

Comment:
I don't want to be a stickler, but please show that this is an upstream change or submit a derivative of this theme.